### PR TITLE
Update Mecca neighbourhoods

### PR DESCRIPTION
**Pull request by stepps00**
Fixes https://github.com/whosonfirst-data/whosonfirst-data/issues/1955.

- [x] Update neighbourhood records
- [x] Add new neighbourhood records
- [x] Update locality record
- [x] PIP
- [x] QA
- [x] Validate changes

### CODE REVIEW

**Comment by stepps00** [comment]
Screen Shot 2021-08-16 at 5 11 41 PM

Screenshot of neighbourhood polygons I've hand-curated for Mecca, visualized in QGIS.

File: data/136/102/622/1/1361026221.geojson [hunk 1]
```diff
@@ -72,10 +91,10 @@
...
-    21.46778,
-    39.90472000000001,
-    21.46778
+    39.895378,
+    21.385696,
+    39.996594,
+    21.482957
 ],
-  "geometry": {"coordinates":[39.90472000000001,21.46778],"type":"Point"}
+  "geometry": {"coordinates":[[[39.991444,21.385696],[39.966181,21.402926],[39.962319,21.413713],[39.93271,21.451858],[39.930136,21.45106],[39.926488,21.449662],[39.924772,21.452657],[39.921339,21.458448],[39.915332,21.456851],[39.910611,21.454854],[39.906749,21.453855],[39.904175,21.453256],[39.898382,21.452058],[39.895378,21.461643],[39.89688,21.462641],[39.89924,21.464239],[39.903102,21.469031],[39.906535,21.473024],[39.90868,21.474821],[39.912113,21.476418],[39.921768,21.48101],[39.935446,21.482957],[39.950465,21.455403],[39.953254,21.451309],[39.970418,21.441724],[39.971384,21.443721],[39.972885,21.442523],[39.977927,21.443621],[39.981789,21.442123],[39.98608,21.438329],[39.996594,21.437131],[39.992088,21.418357],[39.990801,21.408469],[39.989835,21.39968],[39.989835,21.397683],[39.990157,21.394686],[39.99123,21.389192],[39.991444,21.385696]]],"type":"Polygon"}
```
Comment (nvkelso) [inline data/136/102/622/1/1361026221.geojson:99]: This is a pretty big area, that also includes an airport? Should the airport have it's own neighbourhood?

**Review by nvkelso — APPROVED** [review]
I spot checked these, LGTM

One comment about airports.

File: data/136/102/622/1/1361026221.geojson (hunk 1 above)
Comment (stepps00) [reply data/136/102/622/1/1361026221.geojson:99]: Yeah, good point.. I'm going to carve out a new geometry for the airport and adjust surrounding geometries. I'll update any campus records, too.

Comment (stepps00) [reply data/136/102/622/1/1361026221.geojson]: Fixed in 6d5f63e

**Comment by stepps00** [comment]
Most recent commit includes PIP work. I'm going to QA these changes a bit more, then merge.

**Pull request merged**